process/block/preprocess/transactions.go

```diff
@@ -456,7 +456,7 @@ func (txs *transactions) CreateAndProcessMiniBlock(sndShardId, dstShardId uint32
 		)
 
 		if err != nil {
-			log.Error(err.Error())
+			log.Debug(err.Error())
```

Comment:
why is this debug?

Reply:
Because of the curl which genereates many txs and many of them would not be processed until the next round of txs generation, there would be a lot of prints in the console with "lower nonce in tx", as there would be many txs with the same nonce in the pool